.github/dependabot.yml

```diff
@@ -40,6 +40,22 @@ updates:
       - dependency-name: github.com/elastic/mito
     open-pull-requests-limit: 2
 
+  # Azure SDKs
+  - package-ecosystem: "gomod"
+    directory: "/"
+    ignore:
+      - dependency-name: "github.com/elastic/beats/v7"
+    schedule:
+      interval: "daily"
+    groups:
+      azure-sdk-for-go:
+        patterns:
+          - "github.com/Azure/azure-sdk-for-go/*"
+      azure-event-hubs-go:
+        patterns:
+          - "github.com/Azure/azure-event-hubs-go/*"
```

Comment:
Rather than introducing a new `package-ecosystem: gomod` block, would it make sense to just update the existing block above with the `groups` and adding to its `allow` list?

Reply:
Yeah, now I see that apm-server has two blocks because each one targets a different directory.

I'm updating the existing block!

Reply:
I just noticed that by adding a `package-ecosystem: gomod`, we could set a [reviewers](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#reviewers) option to direct PRs to our team.

Do you think it's worth adding a second `package-ecosystem: gomod`, or it's better to keep things as simple as possible and just add a `group` for the Azure dependencies?

I mean something like:

```yaml
  - package-ecosystem: "gomod"
    directory: "/"
    schedule:
      interval: "daily"
    reviewers:
      - "elastic/obs-ds-hosted-services"
    groups:
      azure-sdk-for-go:
        patterns:
          - "github.com/Azure/azure-sdk-for-go/*"
      azure-event-hubs-go:
        patterns:
          - "github.com/Azure/azure-event-hubs-go/*"
```

Reply:
That's a good point. I think it would be ideal if the hosted services team were flagged for review, so having a separate block sounds like the way to go after all.

Reply:
Let me try this: I created a block for CSPs, so we can add a group for each CSP and target specific teams.

```yaml
  # Cloud providers' SDK dependencies
  - package-ecosystem: "gomod"
    directory: "/"
    schedule:
      interval: "daily"
    reviewers:
      - "elastic/obs-ds-hosted-services"
    groups:
      azure-sdks:
        patterns:
          - "github.com/Azure/azure-sdk-for-go/*"
          - "github.com/Azure/azure-event-hubs-go/*"
          - "github.com/Azure/go-autorest/*"
          - "github.com/Azure/azure-storage-blob-go/*"
```